plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -202,3 +202,16 @@
     comment on column {{ relation }}.{{ adapter.quote(column_name) if column_dict[column_name]['quote'] else column_name }} is {{ escaped_comment }};
   {% endfor %}
 {% endmacro %}
+
+{%- macro postgres__get_show_grant_sql(relation) -%}
+  select grantee, privilege_type
+  from information_schema.role_table_grants
+      where grantor = current_role
```

Comment:
Not a TODO, just an open question: Should we / do we need to limit ourselves to just the grants provided by the `current_role`? 

To copy-paste what I just said in `dbt-snowflake`:

> The original motivation for this was: If someone else has granted a privilege on this model, and then dbt tries to revoke it, the revocation may fail. But if dbt's user/role owns the table, it should be able to revoke any access on it... right?

By removing this logic, we'd be making a strong claim, that ALL the grants applied on this object, MUST be applied by dbt, in the `grants` config.

No action needed right now. Just wanted to leave the comment for future reference.